File: src/examples/python/users/retrieve_api_keys.py
import os

import easypost
from dotenv import load_dotenv

load_dotenv()
easypost.api_key = os.getenv("EASYPOST_PROD_API_KEY")

# Retrieve child user's API keys
child = easypost.User.retrieve('user_123...')
child_keys = child.all_api_keys()

print(child_keys)

File: src/examples/php/users/retrieve-api-keys.php
<?php

require_once '/Users/jhammond/git/easypost/easypost-tools/vendor/autoload.php';

use EasyPost\EasyPost;
use EasyPost\Error;
use EasyPost\User;
use Dotenv\Dotenv;

$dotenv = Dotenv::createImmutable('/Users/jhammond/git/easypost/easypost-tools');
$dotenv->load();
EasyPost::setApiKey($_ENV['EASYPOST_PROD_API_KEY']);

try {
    // Pass the ID of a child user
    $user = User::retrieve('user_123...');
    $api_keys = $user->api_keys();

    print_r($api_keys);
} catch (Error $exception) {
    echo $exception;
}
